Update crowall.py

kba977 opened the pull request:
Add support for Mac.

kba977 committed:
Update crowall.py

Add support for Mac.

guoruibiao commented:
Firstly, thanks for your work. :heart: 

And have you test it in your Mac?  If so, I will merge it as soon as possible.

Hearing from you, :cake:

kba977 committed:
Update crowall.py

Modify a small error. It works on my mac.
![1](
7xrahm.com1.z0.glb.clouddn.com/屏幕快照_2016-11-05_14.13.18.png)

kba977 commented:
Your welcome~  

Yes, It works on my Mac. : )

here is the screenshot.

![1](http://7xrahm.com1.z0.glb.clouddn.com/屏幕快照_2016-11-05_14.13.18.png)

guoruibiao commented:
:+1:  

Thanks for your working. :smile:
